famapy/metamodels/fm_metamodel/operations/fm_number_of_configurations.py

```diff
@@ -0,0 +1,50 @@
+import math
+
+from famapy.core.operations import ProductsNumber
+
+from famapy.metamodels.fm_metamodel.models import FeatureModel, Feature
+
+
+class FMNumberOfConfigurations(ProductsNumber):
```

Comment:
I'm not really sure about this. First, is unclear if you calculate products or configurations and also, the approximation should be clarified up to certain point ( which error, etc etc)

Reply:
I agree on this. Maybe a different name for the operation?
It calculates the "number of configurations" (no products). However, the "core" only provides the `ProductsNumber` operation. Maybe this particular operation on FM doesn't have to extend any abstract operation, only the interface `Operation`.

On the other hand, the idea is to calculate the exact number. I don't know if it is possible to calculate the exact number using only the FM. Algorithm 4.1 in Section 4 of [this paper](https://dl.acm.org/citation.cfm?id=1753242) proposes an algorithm for that, but it is not straighforward to understand.

The approximation error seems to be also complex to determine because arbitrary cross-tree constraints can exist.

I will exclude this operation from the PR until clarify better.